Intersect.Client/Core/Main.cs

```diff
@@ -209,10 +209,10 @@ private static void ProcessGame()
             if (Globals.NeedsMaps)
             {
                 bool canShowWorld = true;
-                if (MapInstance.Get(Globals.Me.CurrentMap) != null)
+                if (MapInstance.Get(Globals.Me.MapId) != null)
```

Comment:
This call being repeated like this scares me, can we have a `MapInstance.TryGet(Globals.Me.MapId, out var mapInstance)` instead?